.github/workflows/ci_linux.yml

```diff
@@ -0,0 +1,46 @@
+name: CI Linux
```

Comment:
why cannot we this as a new job into cibuild.yml?
otherwise we'll need to duplicate all the common settings and also duplicate check job...

Reply:
It's possible. I think that there will be very few common settings, and when divided into different files, it will be possible:
1) fix CI for each OS separately without requiring a rebuild for another OS.
2) quickly understand which build fails just by looking at the `README.md`

Do you still think, that it should be into cibuild.yml?

Reply:
I tested and added duplicate check job into Linux CI.